boilerplate/fullstack/remix/app/routes/supertokens.auth.$.tsx

```diff
@@ -1,25 +1,24 @@
 import { json } from "@remix-run/node";
 import { ActionFunctionArgs, LoaderFunctionArgs } from "react-router-dom";
-import handleAuthAPIRequest from "../lib/authAPIRequestHandler.js";
-import { createPreParsedRequest } from "../lib/superTokensHelpers.js";
+import { createPreParsedRequest, handleAuthAPIRequest } from "../lib/superTokensHelpers.js";
 
-const handleCall = handleAuthAPIRequest(Response);
+async function handleCall(request: Request) {
+    const preParsedRequest = createPreParsedRequest(request);
+    return await handleAuthAPIRequest(Response)(preParsedRequest);
```

Comment:
we can make the return of handleAuthAPIRequest(Response) take a remix request, and then create the preparsedrequest in that function itself. This way, you won't need this handleCall function here at all, and can store `handleAuthAPIRequest(Response)` in a variable

Reply:
✔️